bin/rmt-mirror

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require_relative '../config/boot'
+require 'pathname'
+require 'config'
+require 'pp'
+
+require_relative '../lib/rmt'
+require_relative '../lib/rmt/mirror'
+require_relative '../lib/rmt/rpm'
+require_relative '../lib/rmt/downloader'
+
+repository_url = ARGV[0]
+local_path = ARGV[1]
+
+unless (repository_url and local_path)
```

Comment:
Not sure if this is part of this card, but currently you do not validate user input nor catch its exceptions. Calling `./bin/rmt-mirror foo bar` fails with an URI::BadURIError exception and a stacktrace.